### DISCUSSION TITLE
Support for Trino

### DISCUSSION THREAD

**Discussion by anoopknayak** [sqlc-dev/sqlc#3398] [Q&A]
Hi folks, I was wondering if there is support for https://trino.io/ in sqlc. I couldn't see that in the list of engines and hence I assume it is not supported? Could somebody confirm this.

Secondly, how would I go about adding support for trino as an engine for sqlc and what process do I have to follow?

Thanks,

**Comment by shakedlokits** [comment]
I'm also interested in this, it would allow working with Athena over lakehouses🙏

**Comment by asmyasnikov** [comment]
Maybe plugin system can help you https://github.com/sqlc-dev/sqlc/pull/4247. 
This PR needs review from @kyleconroy
